CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Change Log
+## [v4.26.1](https://github.com/plivo/plivo-php/tree/v4.26.1) (2022-03-08)
```

Comment:
1. Add space after line 1.
2. It's a Bug Fix.
3. Suggestion for notes: 
Attribute `answer_method` is set to optional in [Make call API](https://www.plivo.com/docs/voice/api/call/make-a-call/)

Reply:
Done, please check